ci/pipelines/stubs.yml

```diff
@@ -0,0 +1,211 @@
+---
+resource_types:
+  # - name: cf-cli
+  #   type: docker-image
+  #   source:
+  #     repository: nulldriver/cf-cli-resource
+
+  # - name: slack-notification
+  #   type: docker-image
+  #   source:
+  #     repository: cfcommunity/slack-notification-resource
+  #     tag: latest
+
+resources:
+  - name: stubs-pipeline
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/alphagov/pay-ci
+      branch: PP-11459-create-stubs-ecs-pipeline
+      paths:
+        - ci/pipelines/stubs.yml
+
+  - name: stubs-src
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/alphagov/pay-stubs
+      branch: master
+      username: alphagov-pay-ci-concourse
+      password: ((github-access-token))
+      paths:
+        - ./*
```

Comment:
Not sure about this block. Maybe one for @jfharden to verify. I think we should be watching on a release tag, similar to [other microservices](https://github.com/alphagov/pay-ci/blob/master/ci/pipelines/deploy-to-test.yml#L160). We're using [release tags on a merge to master for pay-stubs too](https://github.com/alphagov/pay-stubs/actions/runs/6563571278/job/17828136214).

Reply:
Yeah rather than any path on master if you could release on the alpha-release_XXX tag instead (this is only applied after all the in-repo tests take place post merge)

Reply:
The only thing of extra note to this:

In the task which builds and pushes to deploy we need to [parse the release tag](https://github.com/alphagov/pay-ci/blob/master/ci/pipelines/deploy-to-test.yml#L1515), and push it to ECR with the XXX-release tag instead of alpha_release-XXX